src/mon/OSDMonitor.cc

```diff
@@ -380,13 +380,15 @@ void OSDMonitor::on_active()
 	     << dendl;
     mapping_job->abort();
   }
-  C_PrintTime *fin = new C_PrintTime(osdmap.get_epoch());
-  mapping.reset(new OSDMapMapping);
-  mapping_job = mapping->start_update(osdmap, mapper,
-				      g_conf->mon_osd_mapping_pgs_per_chunk);
-  dout(10) << __func__ << " started mapping job " << mapping_job.get()
-	   << " at " << fin->start << dendl;
-  mapping_job->set_finish_event(fin);
+  if (g_conf->mon_osd_prime_pg_temp) {
```

Comment:
this fixes http://tracker.ceph.com/issues/19020 also, as src/test/erasure-code/test-erasure-code.sh sets mon_osd_prime_pg_temp to false.

could you add a "Fixes" tag in the commit message?